wasm/lib/src/vm_class.rs

```diff
@@ -27,7 +26,21 @@ pub(crate) struct StoredVirtualMachine {
 
 impl StoredVirtualMachine {
     fn new(id: String, inject_browser_module: bool) -> StoredVirtualMachine {
-        let mut vm: VirtualMachine = Default::default();
+        let mut vm: VirtualMachine = VirtualMachine::new(PySettings {
```

Comment:
In this case, I would rather construct a mutable default settings object, and flip the `initialize_as_external` value to false. This would ease the addition of extra settings in the future.